multimodal/src/autogluon/multimodal/utils/environment.py

```diff
@@ -7,6 +8,11 @@
 import torch
 from torch import nn
 
+try:
+    from mmcv.parallel import DataContainer
+except:
+    pass
```

Comment:
Should we do `pass` or set mmcv to `None`?

Reply:
We don't use `mmcv` in this file. I think it doesn't matter setting None or not.